### PR TITLE
Wyoming integration docs

### PR DESCRIPTION
**Pull request by synesthesiam**
## Proposed change

Stub docs for Wyoming integration. Includes links to Whisper/Piper add-ons via redirects.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_redirects [hunk 1]
```diff
@@ -382,6 +382,8 @@
 /addons/snips https://github.com/home-assistant/hassio-addons/blob/master/snips/README.md
 /addons/ssh https://github.com/home-assistant/hassio-addons/blob/master/ssh/README.md
 /addons/tellstick https://github.com/home-assistant/hassio-addons/blob/master/tellstick/README.md
+/addons/whisper https://github.com/home-assistant/hassio-addons/blob/master/whisper/README.md
```
Comment (balloob) [inline source/_redirects]: Should this be My links to install it directly

File: source/_integrations/wyoming.markdown [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ha_category:
+  - Voice
+ha_iot_class: Local Push
+ha_release: '2023.5'
+ha_codeowners:
+  - '@balloob'
+  - '@synesthesiam'
+ha_domain: wyoming
+ha_integration_type: integration
+ha_quality_scale: internal
```
Comment (balloob) [inline source/_integrations/wyoming.markdown]: It’s not internal

File: source/_integrations/wyoming.markdown [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+The Wyoming integration connects external voice services to Home Assistant using a [small protocol](https://github.com/rhasspy/rhasspy3/blob/master/docs/wyoming.md). This enables [Assist](/docs/assist) to use a variety of local [speech-to-text](/integrations/stt/) and [text-to-speech](/integrations/tts/) systems, such as:
+
+* [Whisper](/addons/whisper)
```
Comment (balloob) [inline source/_integrations/wyoming.markdown]: Let’s explain more what it is maybe. (Or maybe in second PR)

**Review by balloob — DISMISSED** [review]

File: source/_integrations/wyoming.markdown (hunk 2 above)
Comment (synesthesiam) [reply source/_integrations/wyoming.markdown]: What should it be?

File: source/_integrations/wyoming.markdown (hunk 3 above)
Comment (synesthesiam) [reply source/_integrations/wyoming.markdown]: Will do in a later PR

File: source/_integrations/wyoming.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/wyoming.markdown]: None at this point

```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks, @synesthesiam 👍 

../Frenck

**Pull request merged**